pootle/apps/pootle_store/models.py

```diff
@@ -1280,16 +1280,27 @@ def create(self, *args, **kwargs):
         if "filetype" not in kwargs:
             filetypes = ProjectFiletypes(kwargs["translation_project"].project)
             kwargs['filetype'] = filetypes.choose_filetype(kwargs["name"])
+        tp = kwargs["translation_project"]
+        if tp.language == tp.project.source_language:
```

Comment:
it would be probs be better, but im not entirely clear if this is the best logic

```
    def get_template_translationproject(self):
        """Returns the translation project that will be used as a template                                                                                                                          
        for this project.                                                                                                                                                                           
                                                                                                                                                                                                    
        First it tries to retrieve the translation project that has the                                                                                                                             
        special 'templates' language within this project, otherwise it                                                                                                                              
        falls back to the source language set for current project.                                                                                                                                  
        """
        try:
            return self.translationproject_set.get(language__code='templates')
        except ObjectDoesNotExist:
            try:
                return self.translationproject_set \
                           .get(language=self.source_language_id)
            except ObjectDoesNotExist:
                pass
```